### ISSUE TITLE
Cannot setup new Metabase instance, when browser locale is different from available language

### ISSUE THREAD

**Issue by martinnormark** [metabase/metabase#13045]
**Describe the bug**
`Let's get started` button not working when browser locale is not supported.

**Logs**
```
setup:1 Refused to load the script 'data:application/javascript;base64,KGZ1bmN0aW9uKCkgewoJLy8gaHR0cHM6Ly9kZXZlbG9wZXJzLmdvb2dsZS5jb20vYW5hbHl0aWNzL2Rldmd1aWRlcy9jb2xsZWN0aW9uL2FuYWx5dGljc2pzLwoJdmFyIG5vb3BmbiA9IGZ1bmN0aW9uKCkgewoJCTsKCX07Cgl2YXIgbm9vcG51bGxmbiA9IGZ1bmN0aW9uKCkgewoJCXJldHVybiBudWxsOwoJfTsKCS8vCgl2YXIgVHJhY2tlciA9IGZ1bmN0aW9uKCkgewoJCTsKCX07Cgl2YXIgcCA9IFRyYWNrZXIucHJvdG90eXBlOwoJcC5nZXQgPSBub29wZm47CglwLnNldCA9IG5vb3BmbjsKCXAuc2VuZCA9IG5vb3BmbjsKCS8vCgl2YXIgdyA9IHdpbmRvdywKCQlnYU5hbWUgPSB3Lkdvb2dsZUFuYWx5dGljc09iamVjdCB8fCAn...0gY2F0Y2ggKGV4KSB7CgkJfQoJfTsKCWdhLmNyZWF0ZSA9IGZ1bmN0aW9uKCkgewoJCXJldHVybiBuZXcgVHJhY2tlcigpOwoJfTsKCWdhLmdldEJ5TmFtZSA9IG5vb3BudWxsZm47CglnYS5nZXRBbGwgPSBmdW5jdGlvbigpIHsKCQlyZXR1cm4gW107Cgl9OwoJZ2EucmVtb3ZlID0gbm9vcGZuOwoJd1tnYU5hbWVdID0gZ2E7CgkvLyBodHRwczovL2dpdGh1Yi5jb20vZ29yaGlsbC91QmxvY2svaXNzdWVzLzMwNzUKCXZhciBkbCA9IHcuZGF0YUxheWVyOwoJaWYgKCBkbCBpbnN0YW5jZW9mIE9iamVjdCAmJiBkbC5oaWRlIGluc3RhbmNlb2YgT2JqZWN0ICYmIHR5cGVvZiBkbC5oaWRlLmVuZCA9PT0gJ2Z1bmN0aW9uJyApIHsKCQlkbC5oaWRlLmVuZCgpOwoJfQp9KSgpOw==' because it violates the following Content Security Policy directive: "script-src 'self' 'unsafe-eval' https://maps.google.com https://apis.google.com https://www.google-analytics.com https://*.googleapis.com *.gstatic.com  'sha256-lMAh4yjVuDkQ9NqkK4H+YHUga+anpFs5JAuj/uZh0Rs=' 'sha256-sMNbXyc1lLzhHbH/CKs11HIQMnMkZAN2eA99WhJeEC0=' 'sha256-JJa56hyDfUbgNfq+0nq6Qs866JKgZ/+qCq2pkDJED8k='". Note that 'script-src-elem' was not explicitly set, so 'script-src' is used as a fallback.

react_devtools_backend.js:12513 Uncaught TypeError: Cannot read property 'defaultLanguage' of null
    at t.value (app-main.bundle.js?46762e85f5bd9f1e9de3:5)
    at c._renderValidatedComponentWithoutOwnerOrContext (vendor.bundle.js?46762e85f5bd9f1e9de3:1)
    at c._renderValidatedComponent (vendor.bundle.js?46762e85f5bd9f1e9de3:1)
    at c._updateRenderedComponent (vendor.bundle.js?46762e85f5bd9f1e9de3:1)
    at c._performComponentUpdate (vendor.bundle.js?46762e85f5bd9f1e9de3:1)
    at c.updateComponent (vendor.bundle.js?46762e85f5bd9f1e9de3:1)
    at c.receiveComponent (vendor.bundle.js?46762e85f5bd9f1e9de3:1)
    at Object.receiveComponent (vendor.bundle.js?46762e85f5bd9f1e9de3:1)
    at Object.receiveComponent (react_devtools_backend.js:12538)
    at Object.object.<computed> [as receiveComponent] (react_devtools_backend.js:12254)
value @ app-main.bundle.js?46762e85f5bd9f1e9de3:5
_renderValidatedComponentWithoutOwnerOrContext @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
_renderValidatedComponent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
_updateRenderedComponent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
_performComponentUpdate @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
updateComponent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
receiveComponent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
receiveComponent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
receiveComponent @ react_devtools_backend.js:12538
object.<computed> @ react_devtools_backend.js:12254
_updateRenderedComponent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
_performComponentUpdate @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
updateComponent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
receiveComponent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
receiveComponent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
receiveComponent @ react_devtools_backend.js:12538
object.<computed> @ react_devtools_backend.js:12254
_updateRenderedComponent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
_performComponentUpdate @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
updateComponent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
receiveComponent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
receiveComponent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
receiveComponent @ react_devtools_backend.js:12538
object.<computed> @ react_devtools_backend.js:12254
_updateRenderedComponent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
_performComponentUpdate @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
updateComponent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
performUpdateIfNecessary @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
performUpdateIfNecessary @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
performUpdateIfNecessary @ react_devtools_backend.js:12501
object.<computed> @ react_devtools_backend.js:12254
b @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
perform @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
perform @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
perform @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
M @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
closeAll @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
perform @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
batchedUpdates @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
batchedUpdates @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
dispatchEvent @ vendor.bundle.js?46762e85f5bd9f1e9de3:1
Show 11 more frames

```

**To Reproduce**
1. Install new instance of metabase using Docker
2. Go to '/setup' page
3. Click on 'Let's get started'
4. Nothing happens

**Expected behavior**
Expected to see the language and user setup elements

**Screenshots**
![image](https://user-images.githubusercontent.com/67565/89036808-a735ad00-d33d-11ea-87a1-cbbc7afc25b8.png)

Chrome language settings:
![image](https://user-images.githubusercontent.com/67565/89036832-b583c900-d33d-11ea-923f-eda657a34e87.png)

**Workaround**
This can be worked around by changing the language priority in Chrome, so a supported locale is at the top:
![image](https://user-images.githubusercontent.com/67565/89037167-4fe40c80-d33e-11ea-95ed-6754c214f923.png)

Clicking `Let's get started` now works:
![image](https://user-images.githubusercontent.com/67565/89037203-60948280-d33e-11ea-96d5-7d236b1070c6.png)

**Information about your Metabase Installation:**

You can get this information by going to Admin -> Troubleshooting.

- Your browser and the version: Chrome, 84.0.4147.105
- Your operating system: Windows 10.2004 (Build 19041.421)
- Your databases: MS SQL
- Metabase version: v0.36.1
- Metabase hosting environment: Docker
- Metabase internal database: H2

**Severity**
Blocking setup progress, unless you figure out the workaround.

**Additional context**
N/A

**Comment by flamber** [comment]
Hi @martinnormark Tak for rapportering :-)
Hmmm... one would think that I should have caught this, but none of my browsers have Danish as primary locale. Reproduced.

**Comment by flamber** [comment]
Fixed in 0.36.2 - released in a bit - tak for hjælpen ;-)

**Comment by martinnormark** [comment]
@flamber Mange tak 👍🏻

Kudos for quick turnaround!
